### PR TITLE
NeatFeats

### LINKED ISSUE #55: Player notification for new players
Give players a DM to join Discord and how to join Discord. I.e., they will see this in their screen **if they are not linked to Discord**:
`Hello <playername>! Thank you for joining the AwF community. You may join Discord on awf.yt and likn yourself with /linkme`
There was an existing implementation of this, but it has since not been added back when things were modified with #43.

### LINKED ISSUE #62: Modernize all commands with pinging of servers
The `?starts`, `?stops` and maybe other commands do not support pinging of servers, i.e. #general. This would be a nice feature to have, as moderators wouldn't need to type in the server's folder name correctly anymore.

### LINKED ISSUE #63: Manual player linking for moderators to use
Since players can link themselves with `/linkme` in-game, it would be nice for moderators/admins to have the ability of using `?linkFactorio`, `?unlinkFactorio` or other such commands to link and unlink players respectively.

### PR DESCRIPTION
**Pull request by oof2win2**
## Resolves #63
Adds two commands for manual player linking by admins
## Resolves #62
`?starts` and `?stops` allow for pinging servers, i.e. `#awf-regular`
## Resolves #55 
When a player without roles in the database joins, they are automatically invited to join the AwF.yt community - the message can be edited in `config/messages.js`

### CODE REVIEW

File: config/messages.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "firstJoinMessage": "Heyas! Welcome to the AwF community. Since you are new, you don't have many permissions. For those, please go to the awf.yt Discord"
```
Comment (DistroByte) [inline config/messages.json:2]: perhaps add a link itself, make it more obvious

File: events/guild/message.js [hunk 2]
```diff
@@ -5,11 +5,6 @@ module.exports = async (client, message) => {
     let args = message.content.slice(prefix.length).trim().split(/ +/g);
     let cmd = args.shift().toLowerCase();
     if (message.author.bot) return;
-    if (message.content.startsWith(prefix)) {
```
Comment (DistroByte) [inline events/guild/message.js:8]: not sure but I think you removed command handler here

Comment (DistroByte) [reply events/guild/message.js:8]: Oh actually sorry, didnt see whole change, resolved

File: config/messages.json (hunk 1 above)
Comment (oof2win2) [reply config/messages.json]: Changed to http://awf.yt

**Review by DistroByte — APPROVED** [review]

**Pull request merged**